.github/workflows/build_and_test.yml

```
@@ -44,6 +44,8 @@ jobs:
       - name: Set up Conda environment
         uses: mamba-org/setup-micromamba@v1
         with:
+          create-args:
+            python=${{ matrix.python-version }}
```

Feels like it should have the `>-` thing (apparently the aim is to have a multi-line value)

```suggestion
          create-args: >-
            python=${{ matrix.python-version }}
```

---

Ok